ygot/diff.go

```diff
@@ -215,6 +215,12 @@ func findSetLeaves(s GoStruct) (map[*pathSpec]interface{}, error) {
 			return
 		}
 
+		// Handle the case of having an annotated struct - in the diff case we
```

Comment:
Should we mention this in the comment on the public func? (Diff)

Reply:
Yes, SGTM. Done.